apps/populate-timeline.test.ts

```diff
@@ -268,7 +268,7 @@ describe("when there are some workflows which haven't been added as cases", () =
 describe("when there are some workflows which have been added as cases", () => {
   beforeAll(async () => {
     mockFindMany.mockResolvedValue([mockWorkflow])
-    ;(fetch as jest.Mock).mockClear()
+    ;(fetch as unknown as jest.Mock).mockClear()
```

Comment:
I've added the type definitions for `node-fetch` and for some reason can't cast `fetch` as `jest.Mock` directly.

Reply:
Nope, because it's a different fetch. That one is on the frontend and so is using the built-in browser fetch. Fetch on the nodejs environment uses `node-fetch` at the moment which is the same API but different implementation.